_articles/linux-terminal_for-beginners.md

```diff
@@ -26,25 +26,38 @@ A box like the one below will appear.
 
 When you run a command with **sudo** in front of it, You'll be prompted for your password, but when you type it, it won't show the letters as you type in the terminal, but type your password anyway and press <kbd>enter</kbd>.
 
-#### Frequently Used Commands
+#### Frequently Used Commands:
+*Omit the brackets when entering the command.*
+
+This series of commands will update the sources.list (a list telling your Ubuntu Operating System where to look for software and updates.)
```

Comment:
`apt update` will update the package database that holds the names of all packages and their versions _from_ the remote repositories listed in sources.list.  sources.list isn't updated at all with this command.